Assets/Scripts/AdsAndIAP/AdsManager.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
```

Comment:
Can you add a header here including `Author` and `Description` (see other scripts for the comment format)